Expensify/App
# Add Emoji Picker

CC: @roryabraham since I'm using the modal you made
### Details
This is a MVP for the emoji picker, it is just a basic picker that works on all platforms without features such as skin tone and a category jumper

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/143023

### Tests/QA
1. Hover over the emoji icon and make sure its style changes
2. Click the icon and make sure an emoji picker pops up
3. Scroll down and make sure the category headers are sticky
4. Click an emoji and make sure it is added to the text field
5. Open the picker again
6. On web/desktop you should see a search bar, on mobile you should not
7. On web/desktop search for something
6. Make sure your search term is reflected in the list

### Tested On


### Screenshots

![2021-04-09_12-40-32](https://user-images.githubusercontent.com/42391420/114236641-26ed3000-993f-11eb-8f9c-fc00f11ad64c.png)
![2021-04-09_12-24-03](https://user-images.githubusercontent.com/42391420/114236649-28b6f380-993f-11eb-8b25-7c38c38de06b.png)



https://user-images.githubusercontent.com/42391420/112886416-27f4a680-908f-11eb-90b6-e0b5411c4b9f.mp4
